packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -135,6 +135,12 @@ export interface IBasePickerProps<T> extends React.Props<any> {
    * A callback to override the default behavior of adding the selected suggestion on dismiss.
    */
   onDismiss?: (ev?: any, selectedItem?: T) => void;
+  /**
+   * Adds an additional alert for the currently selected suggestion. This prop should be set to true for IE11 and below, as it
+   * enables proper screen reader behavior for each suggestion. It should not be set for modern browsers (Edge, Chrome).
+   * @default false
+   */
+  enableSelectedSuggestionAlert?: boolean;
```

Comment:
Doesn't this essentially make it a breaking change for IE 11? Since by default it's false, anything that's consuming it won't  know to have a check for IE11 and won't pass through the variable, so screen readers won't read it properly. @dzearing and @cliffkoh should the picker check for ie 11 itself maybe?

Reply:
It does make it a breaking change for IE 11, but it also fixes it for Chrome and Edge. 

Do we have browser detection logic in fabric?

Reply:
I think it's worth the breaking change, because this makes host apps aware of the issue.